bower.json

```diff
@@ -21,6 +21,6 @@
         "spec"
     ],
     "dependencies": {
-        "moment": "~2.10.3"
+        "moment": "~2.12"
```

Comment:
I like the idea of making this `2.*` given the semver discussion.  Let's do that.